README.md

```diff
@@ -38,6 +38,8 @@ kind: Disruption
 metadata:
   name: node-failure
   namespace: chaos-demo # it must be in the same namespace as targeted resources
+  annotations:
+    chaos.datadoghq.com/environment: "dev" # must match the environment you are running your disruption in. The options here are controlled by the operators of the chaos-controller
```

Comment:
I agree with Matthieu's comment to try without this if the error message is specific enough to guide the user, OR to be more specific in the comment because it sounds to me that:
1. it is mandatory everywhere, every time
2. users will reach out to know what they are supposed to put in there (env being generic and they'll likely not check the controller settings in all envs)